server/liveblog/themes/themes.py

```diff
@@ -25,6 +27,9 @@ class ThemesResource(Resource):
             'type': 'string',
             'unique': True
         },
+        'zippedfiles': {
```

Comment:
Is there a reason to keep this file associated in the database ?
Can't we just extract it and remove it ?